BoardingPass/BoardingInformation.swift

```diff
@@ -80,11 +86,14 @@ public extension BoardingInformation {
 
 private  extension UINavigationController {
 
-    func viewControllerBefore(viewController: UIViewController) -> UIViewController? {
-        guard let index = viewControllers.indexOf(viewController) where index > 0 else {
+    func viewController(beforeController viewController: UIViewController) -> UIViewController? {
+        guard let index = viewControllers.index(of: viewController) else {
+            return nil
+        }
+        guard index > 0 else {
             return nil
         }
-        return viewControllers[index.predecessor()]
+        return viewControllers[(index - 1)]
```

Comment:
#pp `return viewControllers[index.advanced(by: -1)] `
totally a personal style preference, I've been using those for index manipulation